### ISSUE TITLE
Return NS records from digitalocean_domain

### ISSUE THREAD

**Issue by glenjamin** [terraform-providers/terraform-provider-digitalocean#59]
### Terraform Version
```
Terraform v0.11.1
+ provider.digitalocean v0.1.2
```

### Affected Resource(s)
- `digitalocean_domain`

### Expected Behavior
domain should export generated NS records

### Actual Behavior
domain doesn't export any generated NS records

**Comment by atif089** [comment]
+1

Currently, when a domain is added, the 3 default NS records and an A record are generated by DO. There should be a way to remove these.

**Comment by nwbt** [comment]
I agree ... **A** record should not automatically create three **NS** records. It doesn't do this through the UI and seems like it shouldn't through the terraform provider either ... minor detail but worth addressing.

```
Terraform v0.11.10
+ provider.digitalocean v1.0.2
```
